### PR TITLE
EG-2900: Issues in deployment section of Operations Guide

### PR DESCRIPTION
**Pull request by AMcFarlandCampbellR3**

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ This is a sample `node.conf` which details a configuration connecting to the Cor
 
 ```
 
-[nodefull.conf](https://github.com/corda/enterprise/blob/release/ent/4.5/docs/source/resources/nodefull.conf)
+[nodefull.conf](../../resources/nodefull.conf)
```
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md:204]: I wonder if you want to sign-post these a bit more - they look a bit lost on the screen under the code snippets.

File: content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md [hunk 2]
```diff
@@ -303,7 +303,7 @@ networkParametersPath = network-parameters // The network-parameters file is exp
 
 ```
 
-[bridge.conf](https://github.com/corda/enterprise/blob/release/ent/4.5/docs/source/resources/bridge.conf)
+[bridge.conf](../../resources/bridge.conf)
```
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md:306]: I wonder if you want to sign-post these a bit more - they look a bit lost on the screen under the code snippets. (this one and the nodefull.conf link above)

File: content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md [hunk 3]
```diff
@@ -428,8 +428,7 @@ refresh_pattern (Release|Packages(.gz)*)$      0       20%     2880
 refresh_pattern .               0       20%     4320
 debug_options ALL,3
 ```
-
-{{/* github src='docs/source/resources/squidconfig.conf' url='https://github.com/corda/enterprise/blob/release/ent/4.5/docs/source/resources/squidconfig.conf' raw='https://raw.githubusercontent.com/corda/enterprise/release/ent/4.5/docs/source/resources/squidconfig.conf' start='' end='' */}}[squidconfig.conf](https://github.com/corda/enterprise/blob/release/ent/4.5/docs/source/resources/squidconfig.conf)
+[squidconfig.conf](../../resources/squidconfig.conf)
```
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md:431]: Another one to consider giving a bit of context to.

File: content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md [hunk 4]
```diff
@@ -488,7 +487,7 @@ Mar 13 18:44:10 corda-firewall-proxies squid[14261]: Squid Parent: (squid-1) pro
 
 ```
 
-{{/* github src='docs/source/resources/access.conf' url='https://github.com/corda/enterprise/blob/release/ent/4.5/docs/source/resources/access.conf' raw='https://raw.githubusercontent.com/corda/enterprise/release/ent/4.5/docs/source/resources/access.conf' start='' end='' */}}[access.conf](https://github.com/corda/enterprise/blob/release/ent/4.5/docs/source/resources/access.conf)
+[access.conf](../../resources/access.conf)
```
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md:490]: And here again - this is the last time I will call it out - but something like [View access.conf file] (URL). Up to you. def not a showstopper.

File: content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md (hunk 1 above)
Comment (AMcFarlandCampbellR3) [reply content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md:204]: In an ideal world, yes. However, this is how they are all done. I don't have time to fix eveything.

File: content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md (hunk 2 above)
Comment (AMcFarlandCampbellR3) [reply content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md:306]: See comment above.

File: content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md (hunk 3 above)
Comment (AMcFarlandCampbellR3) [reply content/en/docs/corda-enterprise/4.5/node/deploy/env-prod-test.md:431]: And above.

**Pull request merged**